lib/LocalEnsemble.js

```diff
@@ -99,17 +101,20 @@ function LocalEnsemble(ensembleOrModels, connection, maxModels) {
    */
   var i, fillEnsembleInfo, fillModelsInfo, fillSeparateModelsInfo, fill,
     addModel, self = this, ensemble, resource, filename, storageError,
-    localModel;
+    localModel, fillModelsBatch;
   if (!maxModels) {
     maxModels = MAX_MODELS;
   }
+  if (!maxParallelDownloads) {
```

Comment:
shouldn't this be something like: `if (!maxParallelDownloads || maxParallelDownloads > MAX_PARALLEL_DOWNLOADS)`? If so, this remark would apply to line #105 as well

Reply:
It depends. As I added it, the maxParallelDownloads can be set higher than MAX_PARALLEL_DOWNLOADS and it's the programmer's responsibility if it does not work then. As you suggest, MAX_PARALLEL_DOWNLOADS is not a default but a hard limit. We can do that too, but then we might want to find out which is the highest limit that we can afford, right? Maybe 10 is a bit low...